modules/ROOT/pages/administration/access-control/manage-privileges.adoc

```diff
@@ -199,10 +199,149 @@ The following image shows the hierarchy between different graph privileges:
 
 image::privileges_hierarchy.svg[title="Graph privileges hierarchy"]
 
+[[access-control-list-supported-privileges]]
+== Listing supported privileges
+
+Supported privileges can be displayed using the `SHOW SUPPORTED PRIVILEGES` command.
+This lists the privileges that are possible to grant or deny on this server, together with the structure of the privilege.
+
+.Show supported privileges command syntax
+[cols="<15s,<85"]
+|===
+
+| Command
+m| +SHOW SUPPORTED PRIVILEGES+
+
+| Syntax
+a|
+[source, syntax, role="noheader", indent=0]
+----
+SHOW SUPPORTED PRIVILEGE[S]
+  [YIELD { * \| field[, ...] } [ORDER BY field[, ...]] [SKIP n] [LIMIT n]]
+  [WHERE expression]
+  [RETURN field[, ...] [ORDER BY field[, ...]] [SKIP n] [LIMIT n]]
+----
+| Description
+| List all privileges supported by the server.
+
+|===
+
+When using the `RETURN` clause, the `YIELD` clause is mandatory and must not be omitted.
+
+Results will include multiple columns describing the privileges:
+
+[options="header", width="100%", cols="4m,6a,2m"]
+|===
+| Column | Description | Type
+
+| action
+| The privilege action
+| STRING
+
+| qualifier
+| Qualifier to further limit the target of the privilege (`function`, `label`, `procedure`, `property`, `setting`, `username`) or null if not applicable
+| STRING
+
+| target
+| Target of the privilege: `dbms`, `database` or `graph`
+| STRING
+
+| scope
+| List of possible scopes for the privilege (`elements`, `nodes`, `relationships`) or null if not applicable
+| LIST OF STRING
```

Comment:
for all of these, do we want to have something that explains what for example having a property qualifier means?

Reply:
I don't know how to write that in a good way

Reply:
Maybe have it like a table with examples?

```
| scope | example |
--------------------
| elements | GRANT ... ON GRAPH * ELEMENTS * TO role_name |
| nodesWithProperty | GRANT ... ON GRAPH * NODE A (foo IS NOT NULL) TO role_name | // or whatever it'll be named and decided syntax (this will have to be added later once that is in anyway)
...
```

```
| qualifier | example |
--------------------
| function | GRANT EXECUTE FUNCTION abc* ON ... |
| property | GRANT READ {foo} ON ... |
...
```